pkgs/development/python-modules/howdoi/default.nix

```diff
@@ -50,12 +50,14 @@ buildPythonPackage rec {
     export HOME=$(mktemp -d)
   '';
 
-  disabledTests = [ "test_colorize" ];
+  disabledTests = [
+    "test_colorize"
+    "test_missing_pre_or_code_query"
+  ];
```

Comment:
We should add a short comment why those are disabled

Reply:
@SuperSandro2000 i have raised a fix and already merged to the upstream but version not bumped, should we update the reference to the latest commit or keep it to use the tag with this issue?

Reply:
Use `patches` to apply the commit to the current version.